dashboard/app/reporting.go

```diff
@@ -457,7 +457,10 @@ func createBugReport(c context.Context, bug *Bug, crash *Crash, crashKey *db.Key
 			rep.Maintainers = append(rep.Maintainers, mgr.CC.BuildMaintainers...)
 		}
 	}
-	if job != nil {
+	// Note that we additionally check if the bisect result is unreliable. Otherwise we
+	// can eventually Cc people from the unreliable bisection later (e.g. when we did
+	// a bisection using a syz repro and then we are notifying about a C repro).
+	if job != nil && !job.isUnreliableBisect() {
```

Comment:
Maybe it's better to not assign `job = job1` in the beginning  of the function for unreliable bisects? Then we also won't override the crash with the bisect crash (which may be older).

If we have `bug.BisectCause == BisectHorizont` (happens on the oldest tested release), then isUnreliableBisect should return true b/c it should have `job.Flags&BisectResultRelease` set. This is probably right.

But if we have `bug.BisectCause == BisectInconclusive` (bisected to a range of commits), does isUnreliableBisect return true? I think isUnreliableBisect can still return false for such bisections.
We won't add to CC in this case b/c bisectFromJob only adds to CC when bisection has only 1 commit (but BisectInconclusive means more than 1 commit). However, we will still show all these commits in the email. I am thinking if this is the right thing to do.

Reply:
> Maybe it's better to not assign job = job1 in the beginning of the function for unreliable bisects? 

That sounds reasonable. Thanks, I updated the PR.

> But if we have bug.BisectCause == BisectInconclusive (bisected to a range of commits), does isUnreliableBisect return true?

Judging by the source code, it will return false
https://github.com/google/syzkaller/blob/fcfad4ffcf3aa3ecced8298f5816649b800ecd26/dashboard/app/entities.go#L263-L273